src/DynamoCoreWpf/UI/GuidedTour/GuidesManager.cs

```diff
@@ -179,6 +184,29 @@ private HostControlInfo CreateHostControl(Step jsonStepInfo)
             return popupInfo;
         }
 
+        /// <summary>
+        /// This methos will create an StepUIAutomation instance based in the information passed as parameter
```

Comment:
`based on`

Reply:
@QilongTang this comment was updated in the next commit:  https://github.com/DynamoDS/Dynamo/pull/11937/commits/65d76d3883ad2d050273a178016f454345b30ea8